Jenkinsfile

```diff
@@ -142,48 +156,104 @@ pipeline {
                     }
                 }
 
-                stage('IT (Java)') {
+                stage('Integration') {
```

Comment:
Can we rename this to `IT` and `Update Test` to `Update`? Currently, it get's truncated on my relatively large laptop screen (16" macbook pro).
<img width="1211" alt="Screenshot 2020-11-25 at 10 03 35" src="https://user-images.githubusercontent.com/3511026/100205499-7d644f80-2f05-11eb-948c-09036a10cca9.png">

Reply:
truncation? not in my pipeline! :sunglasses: 

but yeah sure :smile:

Reply:
Do you use a 48" OLED TV as monitor? 😆

Reply:
UltraWide :cool:

Reply:
I code in the only acceptable way, in IMAX of course